### ISSUE TITLE
Network: Support CORS enabled images

### ISSUE THREAD

**Issue by dragonzone** [almende/vis#3476]
In the Images class, when creating a new Images, vis.js should add the following attribute to 'img':

```
img.crossOrigin = "Anonymous";
```

In case user wants to save the network that has images from a different domain.

https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image

**Comment by wimrijnders** [comment]
OK I understand what this is about. So a picture from a cross-site needs permission from that server to save an image. Fine.

Can you think of a way to actually test this? An example would be welcome, perhaps a link to go with it.

**Comment by dragonzone** [comment]
Yes, In Chrome debugger, you can view the response header. Below is an example:

![image](https://user-images.githubusercontent.com/10065158/30750882-a1e0ec00-9f85-11e7-9f8a-08ca1916000a.png)

**Comment by wimrijnders** [comment]
Do you have an image link that has this header? Well, of course you do, the screenshot above QED, but can you give me the link as well? For testing purposes.

**Comment by dragonzone** [comment]
Unfortunately, it's an internal server, therefore, there is not public URL. I did image does return allow-origin:

https://lh3.googleusercontent.com/-b7VC94Dniys/AAAAAAAAAAI/AAAAAAAAAAA/ACnBePZC2ZaBHz0GYPvbQT-rUhaoMznjmg/s32-c-mo/photo.jpg
